docs/getting-started-on-snowplow-open-source/quick-start-aws/summary-of-what-you-have-deployed/index.md

```diff
@@ -20,14 +20,24 @@ Note: you can very easily edit the script or run each of the terraform modules i
   </TabItem>
   <TabItem value="snowflake" label="Snowflake">
 
+![](images/snowflake.png)
+
+  </TabItem>
+  <TabItem value="databricks" label="Databricks">
+
+![](images/snowflake.png)
```

Comment:
```suggestion
![](images/databricks.png)
```

Reply:
I need you to make the image for this one!